chia/util/initial-config.yaml

```diff
@@ -214,6 +214,7 @@ pool:
   logging: *logging
   network_overrides: *network_overrides
   selected_network: *selected_network
+  default_max_spends_in_tx: 100
```

Comment:
I put this here to ask if you think we should make this configurable, or leave it hard-coded to 100

Reply:
it looks like this isn't used at all right now, and it's hard coded to 300.
I don't have a strong opinion, I can't judge if configuring the default is ever important (given you can specify an exact limit in the RPC).

Reply:
Thanks, will remove it from the config - that will go in in a separate PR if we do it.